dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -271,23 +431,42 @@ private SolrInputDocument index(Item item) throws SQLException, MetadataBindExce
         return doc;
     }
 
+    private boolean willChangeStatus(Item item) throws SQLException {
+
+        List<ResourcePolicy> policies = authorizeService.getPoliciesActionFilter(context, item, Constants.READ);
+        for (ResourcePolicy policy : policies) {
+            if (policy.getGroup().getName().equals("Anonymous")) {
+                System.out.println("found anonymous READ for item " + item.getID());
+                if (policy.getStartDate() != null && policy.getStartDate().after(new Date())) {
+                    System.out.println("policy has start date after current date: " + policy.getStartDate());
```

Comment:
I'm worried this will get noisy in production. Looks like a debugging print out?